aws/notebooks/helper_functions.py

```diff
@@ -0,0 +1,77 @@
+import random, uuid
+
+def new_job_name_from_config ( dataset_directory, code_choice, 
+                               algorithm_choice, cv_folds,
+                               instance_type, trim_limit = 32 ):
+    job_name = None    
```

Comment:
It would be nice to have docstrings since these are user accessible

Reply:
Done